setup.sh

```diff
@@ -0,0 +1,55 @@
+RED='\033[0;31m'
+YLW='\033[0;33m'
+BLUE='\033[0;34m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+
+echo -e "ADVERTISED_URL: ${BLUE}$1${NC}";
+[ -z "$1" ] && echo "ADVERTISED_URL not set. Enter the host url you would like to advertise." && exit;
+
+while true; do
+    read -p "Do you wish to continue? " yn
+    case $yn in
+        [Yy]* ) break;;
+        [Nn]* ) exit;;
+        * ) echo "Please answer yes or no.";;
+    esac
+done
+
+# comment this for mac OS
+# sed -i .bak '/ADVERTISED_URL/d' ./.env;
+
+sed -i '/ADVERTISED_URL/d' ./.env;
```

Comment:
this command needs to be modified slightly to work on mac. we should find an alternative likely.

possibly, if mac do; if linux do.